### ISSUE TITLE
Screen goes white when loading publish form

### ISSUE THREAD

**Issue by duncanmcclean** [doublethreedigital/runway#36]
## Description

When I try to edit a model in my resource, the screen goes white after a few seconds. Presumably to do with something we're doing with custom publish forms.

## Exact steps to reproduce

1. Go to a resource
2. Click on record to view it
3. Screen goes white

## Environment

```
Statamic 3.1.16 Pro
Laravel 8.43.0
PHP 8.0.3
aryehraber/statamic-captcha 1.5.1
doublethreedigital/duplicator 1.3.2
doublethreedigital/runway 2.0.2
octoper/statamic-html-minify 0.0.5
```

**Comment by 16hands** [comment]
I can also confirm this, and add that rolling back versions doesn't help. I think not being able to roll back could be with composer pulling in a precompiled dist zip.  Deleting vendor may help but there are other issues with older versions anyway 

Its a very useful plugin @duncanmcclean, thank you for your effort with it

**Comment by duncanmcclean** [comment]
You should be able to roll back to pre-2.0 to fix the issue for now. Bear in mind, you'll need to change `'resources'` back to `'models'` in your config when downgrading.

I'm hoping to have some time to look into this evening.

**Comment by duncanmcclean** [comment]
Just tagged a bug fix for this issue, [v2.0.3](https://github.com/doublethreedigital/runway/releases/tag/v2.0.3).

**Comment by 16hands** [comment]
@duncanmcclean Thank you
